_scripts/clone-l10n.sh

```diff
@@ -1,64 +1,104 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
+# ensure the script errors out when a command fails
 set -e
 
-MODULE=$1
+MODULE="$1"
 
-if [ -z "$MODULE" ]; then
-  echo "\nYou need specify a module.\n"
-  exit 0
-fi
+log() {
+    echo "[clone-l10n] $1"
+}
+
+error() {
+    log "$1" >&2
+    exit 1
+}
+
+[ -n "$MODULE" ] || error "You need to specify a module."
 
 if [ -n "$FXA_L10N_SKIP" ]; then
-    echo "Skipping fxa-content-server-l10n update..."
+    log "Skipping fxa-content-server-l10n update..."
     exit 0
 fi
 
-DOWNLOAD_PATH="https://github.com/mozilla/fxa-content-server-l10n.git"
-SCRIPT_DIR=$(dirname "$0")
-MODULE_PATH="$SCRIPT_DIR/../packages/$MODULE"
+DOWNLOAD_PATH="${FXA_L10N_DOWNLOAD_PATH:-fxa-content-server-l10n}"
+SCRIPT_DIR="$(dirname "$0")"
+MODULE_PATH="$(realpath -m "$SCRIPT_DIR/../packages/$MODULE")"
```

Comment:
I think this could be an issue for macOS users since I believe `realpath` isn't available there.

Reply:
AFAICS and as pointed out by @jbuck this script is only used in the context of Docker builds, which means the environment the script runs in is the same and depends on the Docker base image, not the executing host.

But if necessary, I could fall back to BSD `readlink` in case the GNU `realpath` is not available.

Reply:
Fixed, we are now checking for the availability of `realpath` and if not available, fall back to a simple path-building loop (without resolving symlinks - as that is not needed for functionality).